srcpkgs/mesa/template

```diff
@@ -185,11 +185,9 @@ esac
 post_configure() {
 	if [ "$CROSS_BUILD" ]; then
 		find -iname "*.ninja" -exec sed -i "{}" \
+			-e "s;/usr/bin/meson --internal regenerate.*;/usr/bin/touch ${wrksrc}/build/build.ninja;g" \
```

Comment:
Can you test if 557b4a73cf9516f1b58c4c2957e3b782b1724e0b fixed this issue? Otherwise, this PR should be good to ship now

Reply:
> Can you test if [557b4a7](https://github.com/void-linux/void-packages/commit/557b4a73cf9516f1b58c4c2957e3b782b1724e0b) fixed this issue? Otherwise, this PR should be good to ship now

I'm not sure, lemme try push again to make sure.